sdk/node/test/accounts.js

```diff
@@ -0,0 +1,177 @@
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const assert = require('assert')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+const client = new chain.Client()
+const xAccountAlias = `x-${uuid.v4()}`
+const yAccountAlias = `y-${uuid.v4()}`
+
+let mockHsmKey
+
+describe('Accounts test', () => {
+
+  describe('Promise style', () => {
+
+    before('set up API objects', () => {
+
+      // Key and account creation
+      return client.mockHsm.keys.create()
+      .then(key => {
+        mockHsmKey = key
+
+        client.accounts.createBatch([
```

Comment:
instead of just returning right there, I think it may be better to go this route:
```
    before('set up API objects', () => {

      // Key and account creation
      return client.mockHsm.keys.create()
        .then(key => { mockHsmKey = key })
        .then(() => {
          return client.accounts.create({alias: xAccountAlias, rootXpubs: [mockHsmKey.xpub], quorum: 1, tags: {x: 0}})
        })
        .then(() => {
          return client.accounts.create({alias: yAccountAlias, rootXpubs: [mockHsmKey.xpub], quorum: 1, tags: {y: 0}})
        })
    })
```
`createBatch` won't fail if for instance i don't provide a key for one of the accounts, which would then trigger future tests failing for not very clear reasons